src/reference/asciidoc/whats-new.adoc

```diff
@@ -86,6 +87,13 @@ Starting with version 5.1, you must explicitly turn on the `@Publisher` AOP func
 
 See "`<<publisher-annotation>>`" for more information.
 
+[[x5.1-file-writing-message-handler]]
+=== `FileWritingMessageHandler`: New file callback
```

Comment:
This is not a `General Change`, but exactly for `file` module.
So, it must go its own section before or after the mention above in this doc a `FTP and SFTP Changes`.

Reply:
Is it okay if I just remove this lines? I already added a section for it: "`FileWritingMessageHandler`: New file callback", I believe I shouldn't have added it to the `General Change` list.

Reply:
OK. Never mind.
I'll polish this on merge as well and will share here with you what I mean.

Please, fix the Checkstyle violations and we are good to go with merge!

Reply:
Sure, appreciated :)